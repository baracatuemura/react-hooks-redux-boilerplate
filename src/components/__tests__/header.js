import React from "react"
import renderer from "react-test-renderer"

import Header from "../header"

let _nowFn
beforeEach(() => {
  _nowFn = Date.now
  Date.now = jest.fn(() => 1482363367072)
})

afterEach(() => {
  Date.now = _nowFn
})

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
