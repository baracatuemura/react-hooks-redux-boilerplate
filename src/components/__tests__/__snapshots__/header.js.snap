// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<nav
  className="header"
>
  <ul
    className="header__container"
  >
    <li>
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/example"
      >
        ExampleList
      </a>
    </li>
    <li>
      <a
        href="/dynamic-route/1482363367071"
      >
        Dynamic route example
      </a>
    </li>
    <li>
      <a
        href="/about"
      >
        About
      </a>
    </li>
  </ul>
</nav>
`;
